silver/tests/unit/test_payment_method_serializer.py

```diff
@@ -41,12 +41,15 @@ def test_encoding(self):
             'request': request
         })
 
+        self_url_skelet = 'http://testserver/customers/{}/payment_methods/{}/'
```

Comment:
Maybe something like this would help : https://github.com/silverapp/silver/blob/master/silver/tests/spec/test_transactions.py#L40
`skelet` sounds a bit odd.